flowise-embed/src/utils/fileClassificationUtils.ts

```diff
@@ -488,20 +483,16 @@ export const checklistMSDS = `
 • Carga perigosa - Package group - responder em numero romano`;
 
 export const checklistCotacaoDeFrete = `
-• Mercadorias/Commodity
-• INCOTERM - procure os possíveis valores de Incoterm no documento, retorne apenas a sigla
-• Quantidade e tipo de containers
-• Quantidade e tipo de volumes
 • Cubagem - Considere APENAS valores explicitamente demarcados em "m³" ou "m3". NUNCA considere outras informações, como "peso cúbico", ainda que o nome se assemelhe. Caso não encontre valores demarcados em "m³" ou "m3", retorne null. NUNCA preencha com outras informações.
+• INCOTERM -  procure os possíveis valores de Incoterm no documento, retorne apenas a sigla, ou seja, Se o valor do campo for Ex Works, exiba EXW. Se o valor do campo for Free on Board, exiba FOB. Se o valor do campo for Cost, Insurance and Freight, exiba CIF. Se o valor do campo for Cost and Freight, exiba CFR. Se o valor do campo for Free Alongside Ship, exiba FAS. Se o valor do campo for Free Carrier, exiba FCA. Se o valor do campo for Delivered Duty Paid, exiba DDP. Se o valor do campo for Delivered Duty Unpaid, exiba DDU. Se o valor do campo for Carriage Paid To, exiba CPT. Se o valor do campo for Carriage and Insurance Paid To, exiba CIP.
```

Comment:
Sugiro ajustar a pontuação aqui também.
`INCOTERM - Procure os possíveis valores de Incoterm no documento e retorne apenas a sigla. Por exemplo, se o valor do campo for "Ex Works", exiba "EXW". ...`